nixos/modules/services/databases/redis.nix

```diff
@@ -224,6 +229,16 @@ in
 
     environment.systemPackages = [ cfg.package ];
 
+    systemd.services.disable-transparent-huge-pages = {
+      enable = config.services.redis.enable;
```

Comment:
This is already inside a `mkIf config.services.redis.enable`, so no need for the conditional.

Reply:
I want to keep it anyway, because this job will move out of the Redis module eventually.